newIDE/app/src/BehaviorsEditor/index.js

```diff
@@ -103,7 +103,7 @@ const BehaviorsEditor = (props: Props) => {
   );
 
   const windowWidth = useResponsiveWindowWidth();
-  const isSmall = windowWidth === 'small';
+  const isMobileOrTablet = windowWidth === 'small' || windowWidth === 'xsmall';
```

Comment:
I wonder if this logic should be given by the hook:

```js
const { windowWidth, isSmallOrSmaller } = useResponsiveWindowWidth()
```

Reply:
or:
```js
const { windowWidth, isSmallerThan, isBiggerThan } = useResponsiveWindowWidth()
const isSmallDevices = isSmallerThan('small')
```

Reply:
Could be. I have a feeling with have too many sizes now. 
Small/medium/large/xlarge would be fine. I'm not sure about xsmall. A phone is small, not xsmall

Reply:
I've updated the logic to use small only so we probably don't need those functions now